app/models/audit.rb

```diff
@@ -1,8 +1,14 @@
 class Audit < ActiveRecord::Base
-  belongs_to :auditable, polymorphic: true
+  belongs_to :auditable, -> { with_deleted }, polymorphic: true
```

Comment:
Your said this brought some difference to the rest of the system? Some querying?